server/sonar-server/src/test/java/org/sonar/server/plugins/ws/InstalledPluginsWsActionTest.java

```diff
@@ -87,84 +90,117 @@ public void core_plugin_are_not_returned() throws Exception {
+      )
+      );
 
     underTest.handle(request, response);
 
     assertJson(response.outputAsString()).isSimilarTo(
-        "[" +
-            "{" +
-            "  \"key\": \"plugKey\"," +
-            "  \"name\": \"plugName\"," +
-            "  \"version\": \"10\"" +
-            "}" +
-            "]"
-    );
+      "{" +
+        "  \"plugins\":" +
+        "  [" +
+        "    {" +
+        "      \"key\": \"plugKey\"," +
+        "      \"name\": \"plugName\"," +
+        "      \"description\": \"desc_it\"," +
+        "      \"version\": \"10\"," +
+        "      \"license\": \"license_hey\"," +
+        "      \"organization\": " +
+        "      {" +
+        "        \"name\": \"org_name\"," +
+        "        \"url\": \"org_url\"" +
+        "      }, " +
+        "      \"urls\":" +
+        "      {" +
+        "        \"homepage\": \"homepage_url\"," +
+        "        \"issueTracker\": \"issueTracker_url\"" +
+        "      }" +
+        "    }" +
+        "  ]" +
+        "}"
+      );
   }
 
   @Test
   public void plugins_are_sorted_by_name_then_key_and_only_one_plugin_can_have_a_specific_name() throws Exception {
     when(pluginRepository.getMetadata()).thenReturn(
-        of(
-            plugin("A", "name2"),
-            plugin("B", "name1"),
-            plugin("C", "name0"),
-            plugin("D", "name0")
-        )
-    );
+      of(
+        plugin("A", "name2"),
+        plugin("B", "name1"),
+        plugin("C", "name0"),
+        plugin("D", "name0")
+      )
+      );
 
     underTest.handle(request, response);
 
     assertJson(response.outputAsString()).setStrictArrayOrder(true).isSimilarTo(
-        "[" +
-            "{\"key\": \"C\"}" + "," +
-            "{\"key\": \"D\"}" + "," +
-            "{\"key\": \"B\"}" + "," +
-            "{\"key\": \"A\"}" +
-            "]"
-    );
+      "{" +
+        "  \"plugins\":" +
+        "  [" +
+        "    {\"key\": \"C\"}" + "," +
+        "    {\"key\": \"D\"}" + "," +
+        "    {\"key\": \"B\"}" + "," +
+        "    {\"key\": \"A\"}" +
+        "  ]" +
+        "}"
+      );
   }
 
   @Test
   public void only_one_plugin_can_have_a_specific_name_and_key() throws Exception {
     when(pluginRepository.getMetadata()).thenReturn(
-        of(
-            plugin("A", "name2"),
-            plugin("A", "name2")
-        )
-    );
+      of(
+        plugin("A", "name2"),
+        plugin("A", "name2")
+      )
+      );
 
     underTest.handle(request, response);
 
     assertJson(response.outputAsString()).setStrictArrayOrder(true).isSimilarTo(
-        "[" +
-            "{\"key\": \"A\"}" +
-            "]"
-    );
+      "{" +
+        "  \"plugins\":" +
+        "  [" +
+        "    {\"key\": \"A\"}" +
+        "  ]" +
+        "}"
+      );
     assertThat(response.outputAsString()).containsOnlyOnce("name2");
   }
 
   @Test
   public void dash_is_returned_when_version_is_null() throws Exception {
```

Comment:
according to Ruby code, `-` was output in JSON for the version.

This can easily be droped. I agree is does not make sense unless we want to implement a kink of mandatory field.